app/states/unified_database.py

```
@@ -0,0 +1,1208 @@
+"""
+Database manager for unified SQLModels backed by PostgreSQL.
+"""
+
+from typing import List, Dict, Any, Optional, Union
```

Ruff (0.14.7)

`typing.List` is deprecated, use `list` instead

(UP035)